README.md

```diff
@@ -11,6 +11,12 @@ To install `saucectl` using NPM just run:
 $ npm i -g saucectl
 ```
 
+In case you need to download `saucectl` from a known source or run `sauceclt` after installation from a known source, you can use environment variable:
```

Comment:
``sauceclt` typo. Even tho, I'm a bit confused about what does this implies: 
`run `sauceclt` after installation from a known source`. Do we need this second description?

Reply:
The first version with the only mentioning install was correct.